torchsde/_core/misc.py

```diff
@@ -153,3 +153,11 @@ def batch_mvp(m, v):
     mvp = torch.bmm(m, v)  # (batch_size, d, 1)
     mvp = mvp.squeeze(dim=-1)  # (batch_size, d)
     return mvp
+
+
+def grad(inputs, **kwargs):
+    # Workaround for PyTorch bug #39784
+    if torch.is_tensor(inputs):
+        inputs = (inputs,)
+    _inputs = [torch.as_strided(input, (), ()) for input in inputs]
```

Comment:
Another really minor nit: Could you rename `input` variable, since it shadows a builtin function. 

I can merge after this change. Thanks again!

Reply:
Sure thing.